src/config.js

```diff
@@ -14,6 +14,7 @@ export default class RTConfig {
     this.debugMode = false
     this.connectQuery = {}
     this.socketConfigTransform = null
+    this.getAppInfo = null
```

Comment:
`appInfo` is not a good name for that, since the `appInfo` is more related to backendless app API not for RT Client

what about `hostResolver`?